lighthouse-core/audits/preload-lcp-image.js

```diff
@@ -228,6 +243,17 @@ class PreloadLCPImageAudit extends Audit {
     ];
     const details = Audit.makeOpportunityDetails(headings, results, wastedMs);
 
+    // If LCP element was an image and had valid network records (regardless of
+    // if it should be preloaded), it will be found first in the `initiatorPath`.
+    // Otherwise path and length will be undefined.
```

Comment:
nit: Is this describing a potential use case for the debug data? If so I don't think we need it in a comment here.

Reply:
I definitely see your point, but we don't really have another place for it. In theory, this is what debugData is for (it's in the name :), and as someone currently trawling through our various debugDatas, this would be a helpful comment for me rather than following the execution path up and down (in each audit) to figure out what's in it. We don't have `debugData` types (and we shouldn't), so this is the best idea I had to document it 🤷 

I was also trying to explain why you could get no image to preload but still get an initiator path for the image LH doesn't recommend you preload.